### PR TITLE
Restructure docs

### PR DESCRIPTION
**Pull request by probablykasper**
Brings in changes already made in the tauri repo + fixes broken links

Notes:
- I haven't updated the `fr` docs
- Deleted sidebars.json, sidebars.js seems to have replaced it
- `tips/cross-browser.md` was removed. Not sure if that was supposed to happen
- Architecture link in `get-started/intro.md` is broken. Should `ARCHITECTURE.md` be moved into docs, or the link be changed back to a GitHub link?

### CODE REVIEW

**Comment by lucasfernog** [comment]
The CI copies the ARCHITECTURE.md file to this repo, see https://github.com/tauri-apps/tauri/blob/b7a2345b06ca0306988b4ba3d3deadd449e65af9/.github/workflows/update-docs.yml#L79

**Comment by probablykasper** [comment]
Alright, good to know. Otherwise good to go?

**Comment by lucasfernog** [comment]
I need to double check to see if we changed anything that is specific to the next release.

**Comment by lorenzolewis** [comment]
Is there anything I can do to help with this? Would like this to be in place so I can start looking at putting in some more beginner-friendly and architecture guides maybe.

File: docs/en/guides/menu.md [hunk 1]
```diff
@@ -25,6 +25,12 @@ let menu = Menu::new()
   .add_native_item(MenuItem::Copy)
   .add_item(CustomMenuItem::new("hide", "Hide"))
   .add_submenu(submenu);
+// alternatively, using the `with_items` constructor, useful if you end up using conditional compilation
+let menu = Menu::with_items([
+  MenuItem::Copy.into(),
+  CustomMenuItem::new("hide", "Hide").into(),
+  submenu.into(),
+])
```
Comment (FabianLars) [inline docs/en/guides/menu.md]: This is not part of beta.8

File: docs/en/api/config.md [hunk 2]
```diff
@@ -27,8 +27,8 @@ In addition to the JSON defined on the `tauri.conf.json` file, Tauri reads a pla
   The target directory <em>must</em> contain an index.html file.
 </div>`},
 {property: "devPath", type: "string", description: `Can be a path to a folder (either absolute or relative to tauri.conf.json) or a URL (like a live reload server).`},
-{property: "beforeDevCommand", optional: true, type: "string", description: `A command to run before starting Tauri in dev mode.`},
-{property: "beforeBuildCommand", optional: true, type: "string", description: `A command to run before starting Tauri in build mode.`},
+{property: "beforeDevCommand", optional: true, type: "string", description: `A command to run before starting Tauri in dev mode. The PLATFORM, ARCH, FAMILY and PLATFORM_TYPE environment variables are set if you perform conditional compilation.`},
+{property: "beforeBuildCommand", optional: true, type: "string", description: `A command to run before starting Tauri's build pipeline. The PLATFORM, ARCH, FAMILY and PLATFORM_TYPE environment variables are set if you perform conditional compilation.`},
```
Comment (FabianLars) [inline docs/en/api/config.md]: This is not part of beta.8

File: docs/en/guides/updater.md [hunk 3]
```diff
@@ -280,14 +280,14 @@ The *Private key* (privkey) is used to sign your update and should NEVER be shar
 To generate your keys you need to use the Tauri cli.
 
 ```bash
-tauri sign -g -w ~/.tauri/myapp.key
+tauri signer sign -g -w ~/.tauri/myapp.key
```
Comment (FabianLars) [inline docs/en/guides/updater.md]: This is not part of beta.8

File: docs/en/guides/updater.md [hunk 4]
```diff
@@ -280,14 +280,14 @@ The *Private key* (privkey) is used to sign your update and should NEVER be shar
...
 ```
 
 You have multiple options available
 ```bash
 Tauri updates signer.
 USAGE:
-    tauri sign [FLAGS] [OPTIONS]
+    tauri signer sign [FLAGS] [OPTIONS]
```
Comment (FabianLars) [inline docs/en/guides/updater.md]: This is not part of beta.8

File: docs/en/guides/webdriver/example/selenium.md [hunk 5]
```diff
@@ -148,7 +148,7 @@ before(async function() {
   );
 
   const capabilities = new Capabilities();
-  capabilities.set("tauri:options", { application });
+  capabilities.set("tauri:options", { application, args: ["app", "cli", "args"] });
```
Comment (FabianLars) [inline docs/en/guides/webdriver/example/selenium.md]: This is not part of beta.8

File: docs/en/guides/webdriver/example/webdriverio.md [hunk 6]
```diff
@@ -128,6 +128,7 @@ exports.config = {
       maxInstances: 1,
       "tauri:options": {
         application: "../../target/release/hello-tauri-webdriver",
+        args: ["app", "cli", "args"]
```
Comment (FabianLars) [inline docs/en/guides/webdriver/example/webdriverio.md]: This is not part of beta.8

**Review by FabianLars — CHANGES_REQUESTED** [review]
Annotated the next-only changes

**Comment by lucasfernog** [comment]
I was about to check that, thanks @FabianLars faster than me as always :joy:

**Comment by lucasfernog** [comment]
We can just revert the next release changes and merge this PR.

**Comment by probablykasper** [comment]
Alright, think it should be good now

File: docs/en/api/cli.md [hunk 7]
```diff
@@ -29,30 +29,76 @@ This command is pretty helpful when you need to have a quick overview of your ap
...
+
+FLAGS:
+    -a, --api        Initializes a Tauri plugin with TypeScript API.
+    -h, --help       Print help information
+    -V, --version    Print version information
+
+OPTIONS:
+    -d, --directory <directory>      Set target directory for init
+    -n, --name <name>                Name of your Tauri plugin
+    -t, --tauri-path <tauri-path>    Path of the Tauri project to use (relative to the cwd)
```
Comment (FabianLars) [inline docs/en/api/cli.md]: I forgot this one. Thought it was just _renamed_ to `plugin init`, but neither version was part of beta.8

**Review by FabianLars — CHANGES_REQUESTED** [review]

File: docs/en/api/cli.md (hunk 7 above)
Comment (kasper9n) [reply docs/en/api/cli.md]: Ok so I removed the `init plugin` section, was that right?

**Comment by lorenzolewis** [comment]
Is there a way I can help with the merge conflicts here? I don't have write access to the repo and I think my updates this week may have made it a bit of a pain from this version to merge ;)

**Review by FabianLars — APPROVED** [review]

**Comment by FabianLars** [comment]
I hope the cli commands `help` outputs are correct, but nothing too bad there anyway imo. So i'll go ahead and merge so @lorenzolewis can have some more fun lol

**Pull request merged**